Monika After Story/game/script-topics.rpy

```diff
@@ -4860,6 +4860,14 @@ init 5 python:
     addEvent(Event(persistent.event_database,eventlabel="monika_weddingring",category=['romance'],prompt="Engagement ring",random=True))
 
 label monika_weddingring:
+    if not renpy.seen_label("monika_weddingring1"):
+        call monika_weddingring1
+    else:
+        call monika_weddingring2
+    return
```

Comment:
should `return _return` here, since you call other labels that return `derandom`.